plugins/catalog-backend/src/service/types.ts

```diff
@@ -71,6 +74,7 @@ export interface RefreshService {
    * Request a refresh of entities in the catalog.
    */
   refresh(options: RefreshOptions): Promise<void>;
+  refreshByRefreshKeys(options: RefreshByRefreshKeysOptions): Promise<void>;
```

Comment:
What do you think about making this an addition to the existing `RefreshOptions` rather than a new method?

Reply:
If we won't expose the refreshKeys on the DefualtRefreshService we won't need to add the options to the existing RefreshOptions either, so I'll hold on to this and do it based on how we decide.